### PR TITLE
Feat/971: Add link to the make proposal form

### LINKED ISSUE #971: Add link to the make proposal form
https://github.com/vegaprotocol/specs/blob/Add-interface-acceptance-criteria/user-interface/1004-VOTE-vote.md#1004-VOTE-002

Screenshot 2022-08-08 at 14 53 54

### PR DESCRIPTION
**Pull request by sam-keen**
# Related issues 🔗

Closes #971

# Description ℹ️

Provides a link to the governance proposal form from the proposals list

# Demo 📺

Screenshot 2022-08-08 at 15 21 24

Screenshot 2022-08-08 at 15 21 44

### CODE REVIEW

**Review by dexturr — APPROVED** [review]

File: apps/token/src/routes/governance/components/proposals-list/proposals-list.tsx [hunk 1]
```diff
@@ -44,7 +47,16 @@ export const ProposalsList = ({ proposals }: ProposalsListProps) => {
 
   return (
     <>
-      <Heading title={t('pageTitleGovernance')} />
+      <div className="grid xs:grid-cols-2 items-center">
+        <Heading centerContent={false} title={t('pageTitleGovernance')} />
+        <Link
```
Comment (dexturr) [inline apps/token/src/routes/governance/components/proposals-list/proposals-list.tsx:52]: chore (non-blocking): can we add some basic tests to validate this heading and link are rendered?

Comment (sam-keen) [reply apps/token/src/routes/governance/components/proposals-list/proposals-list.tsx:52]: Sure

**Review by MadalinaRaicu — APPROVED** [review]

File: apps/token/src/routes/governance/components/proposals-list/proposals-list.tsx [hunk 2]
```diff
@@ -1,10 +1,13 @@
 import { isFuture } from 'date-fns';
-import { useState } from 'react';
+import React, { useState } from 'react';
```
Comment (notbot00) [inline apps/token/src/routes/governance/components/proposals-list/proposals-list.tsx:2]: question (non-blocking): why is the React import needed here?

**Review by notbot00 — APPROVED** [review]

File: apps/token/src/routes/governance/components/proposals-list/proposals-list.tsx (hunk 2 above)
Comment (sam-keen) [reply apps/token/src/routes/governance/components/proposals-list/proposals-list.tsx:2]: My IDE sometimes does this automatically. I'll remove it the next time I'm working on the project. Thanks

**Pull request merged**